run_fvs.sh

```diff
@@ -6,17 +6,34 @@ date
 pwd
 git status
 
-./build_node.sh
-./create_binary.sh
+docker rm -f host1 || true
+docker run --privileged -v `pwd`:/code --name host1 -tid jpetazzo/dind
+
+docker exec -t host1 bash -c \
+ 'docker -d & \
+ cd /code && \
+ ./build_node.sh && \
+ ./create_binary.sh'
+# docker run --privileged -v `pwd`:/code --name host2 -t jpetazzo/dind bash -c \
+#  'docker -d -D && \
+#  cd /code && \
+#  ./build_node.sh && \
+#  ./create_binary.sh'
+# docker run --privileged -v `pwd`:/code --name host3 -t jpetazzo/dind bash -c \
+#  'docker -d -D && \
+#  cd /code && \
+#  ./build_node.sh && \
+#  ./create_binary.sh'
```

Comment:
I don't really like checking it commented out lines, but I don't feel too strongly about it.